--- origsrc/hyphen-2.8.8/Makefile.am	2013-03-18 05:49:03.000000000 -0500
+++ src/hyphen-2.8.8/Makefile.am	2017-07-23 13:15:59.562345700 -0500
@@ -2,7 +2,7 @@ SUBDIRS= doc tests
 
 lib_LTLIBRARIES = libhyphen.la
 libhyphen_la_SOURCES = hnjalloc.c hyphen.c
-libhyphen_la_LDFLAGS = -version-info 3:0:3
+libhyphen_la_LDFLAGS = -version-info 3:0:3 -no-undefined
 include_HEADERS = hyphen.h
 noinst_HEADERS = hnjalloc.h
 
